src/user/user.service.ts

```diff
@@ -5,8 +5,9 @@ import { Repository } from 'typeorm';
 import { JwtService } from '@nestjs/jwt';
 import { User } from '@src/user/entities/user.entity';
 import { ProfileService } from '@src/profile/profile.service';
-import { IUser } from '@src/user/models/user.interface';
 import { MailService } from '@src/mail/mail.service';
+import * as bcrypt from 'bcrypt';
```

Comment:
"import *" is bad case to use. Try a destructurization case like "import {hash, compare} from 'bcrypt'"